gstchecksumsink.c

```diff
@@ -181,6 +192,9 @@ gst_cksum_image_sink_set_property (GObject * object, guint prop_id,
     case PROP_RAW_OUTPUT:
       checksumsink->dump_output = g_value_get_boolean (value);
       break;
+    case PROP_RAW_LOCATION:
+      checksumsink->raw_file_name = g_strdup (g_value_get_string (value));
```

Comment:
I guess you forgot to freed the string somewhere

Reply:
Freed in gst_cksum_image_sink_stop

Reply:
dah! true... you didn't add a new variable :)

sorry for the noise